src/App.jsx

```diff
@@ -9,32 +11,20 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
 
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${selectedTab.title}`}
+      </h1>
 
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTab.id}
+        onTabSelected={tab => setSelectedTab(tab)}
```

Comment:
```suggestion
        onTabSelected={setSelectedTab}
```

Reply:
use shorthand

Reply:
Can we use such a syntax? (Directly use the parent function)

Reply:
> Can we use such a syntax? (Directly use the parent function)

 you're directly passing the setSelectedTab function reference as the callback. This is possible if the setSelectedTab function itself is designed to accept the tab parameter. This design assumes that the tab parameter will be automatically passed to the function when a tab is selected.